### ISSUE TITLE
HA 2023.4: null in header.classlist

### ISSUE THREAD

**Issue by stellarore** [ben8p/lovelace-tab-redirect-card#6]
I upgraded to home assistant core 2023.4, which broke all of my redirect cards. Each has the same error: 
`
null is not an object (evaluating
'header.classList")
`
Upgrading to 2023.4.1 did not fix it. 

Example Card yaml:

```
type: custom:tab-redirect-card 
redirect:
- user: haadmin 
  entity_ id: timer. dash2 
  entity state: 'idle’
  redirect to tab index: 1

```

**Comment by kwithus** [comment]
Any solution for this so far?

**Comment by alexisspencer** [comment]
also having the same issue here

**Comment by Pirol62** [comment]
Any progress on this?

**Comment by evansalter** [comment]
I opened a PR to fix this: #7 

If anybody is running an older version of HA to test the backward compatibility that woudl be great.

Not sure how quickly this will be merged, so feel free to install the plugin from my forked repo by doing the following:

1. Remove `lovelace-tab-redirect-card` from HACS
2. Add a custom repository in HACS pointing to https://github.com/evansalter/lovelace-tab-redirect-card
3. Empty your browser cache and reload - things should be working

**Comment by Pirol62** [comment]
Already updated. Workes perfectly. Thank you!
